homeassistant/components/atag/config_flow.py

```diff
@@ -1,9 +1,9 @@
 """Config flow for the Atag component."""
-from pyatag import DEFAULT_PORT, AtagException, AtagOne
+import pyatag  # from pyatag import DEFAULT_PORT, AtagException, AtagOne
```

Comment:
Commented code.

Reply:
Woops.. Can I still fix this now that it's merged?

Reply:
No, please make a new PR. Thanks!